docs/laravel-integration.md

```diff
@@ -7,7 +7,7 @@ To follow this guide you will need:
 
 * PHP Installed; this example uses PHP 7.4.
 * [Composer](https://getcomposer.org/download/ ) for dependency management. 
-* [Docker](https://docs.docker.com/get-docker/) for bundling our visualization tools. We have setup instructions for docker on this project's [readme](https://github.com/open-telemetry/opentelemetry-php#development).
```

Comment:
This is actually grammatically correct.  Setup is used as a noun, set up is used as an irregular verb:

https://grammarist.com/spelling/set-up-vs-setup/#:~:text=Set%20Up%20%E2%80%93%20What's%20the%20Difference,set%20up%20the%20computer%E2%80%9D).

Reply:
You're right. Reverted!